projects/detr3d/mmdet3d_plugin/datasets/transforms/transform_3d.py

```diff
@@ -0,0 +1,107 @@
+import mmcv
+import numpy as np
+from mmdet3d.registry import TRANSFORMS
+from numpy import random
+
+
+@TRANSFORMS.register_module()
+class PhotoMetricDistortionMultiViewImage:
```

Comment:
This Aug is general. Maybe you can move this aug into `mmdet/datasets/transoforms/transforms_3d.py` in this PR if you have additional time. If so, you should add UT and type hint for this module.

Reply:
Can we use existing `PhotoMetricDistortion3D` and `MultiviewWrapper` to achieve this function?

Reply:
Sure!